pkg/setting/setting.go

```diff
@@ -144,6 +144,7 @@ var (
 	AdminPassword    string
 	LoginCookieName  string
 	LoginMaxLifetime time.Duration
+	Sigv4AuthEnabled bool
```

Comment:
See my comment above about casing of SigV4.
```suggestion
	SigV4AuthEnabled bool
```